mozilla/blurts-server
# Align navigation with design, remove unused links

# References: 
Jira: MNTOR-1169
Figma: https://www.figma.com/file/gvyyqS9egsGjsH0LUDqT1m/Breach-Resolution-Flow-Design?node-id=214%3A9127&t=sEAfwuHATkggJONz-0



# Description

This splits the left-hand side navigation into a primary and secondary part, and removes unused links. It also adds an icon to the "Resolve data breaches" link, and otherwise updates the styling to more closely match the design.

I encountered another missing font, so I copied that over from Relay and added that as yet another resource to load.

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/218156378-a963146f-5e11-4681-b234-6d6be7a275ee.png)

![Screen Shot 2023-02-10 at 18 25 27](https://user-images.githubusercontent.com/4251/218156473-2b08bb24-3ef0-4c4f-818e-653122f62cdd.png)

# How to test

View the navigation. Resize your screen to make sure it looks good on the relevant screen sizes.

# Checklist (Definition of Done)
- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, layout changes
